cnd/migrations/2020-04-09-052607_create-message-tables/up.sql

```diff
@@ -0,0 +1,133 @@
+-- Your SQL goes here
+
+CREATE TABLE han_ethereum_ether_han_bitcoin_bitcoin_created_swaps
+(
+    id INTEGER               NOT NULL PRIMARY KEY,
+    local_swap_id            UNIQUE NOT NULL,
+    role                     NOT NULL,
+    dial_information         NOT NULL,
```

Comment:
Is the peer id or the address-hint? I cannot be both.

Reply:
Yes, sloppy work by me.  Any preference, we can have both as optional fields?

Reply:
I think we don't need to save the address_hint at all :)

We should save the peer ID because it tells us who we are swapping with.

Reply:
Cool, worked this in.